### PR TITLE
Add workflows to test kube-burner on ppc64le

### LINKED ISSUE #646: [RFE] OCP-CI for ppc64le for release testing
**Is your feature request related to a problem? Please describe.**
There is no release testing CI for kube-burner `ppc64le` binary.

**Describe the solution you'd like**
ppc64le CI for release testing prior publishing the binaries.

### PR DESCRIPTION
**Pull request by kishen-v**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This PR adds workflow to test the kube-burner binary on the ppc64le architecture, as a part of CI/release testing.
A dedicated VM for the project is available, I can work with @krishvoor for the access and details for the broader team's usage.

## Related Tickets & Documents

- Closes #646

### CODE REVIEW

**Comment by krishvoor** [comment]
Hi @kishen-v: Thanks for your PR, can you share your GHA runs from your local fork?

File: .github/workflows/test-k8s-ppc64le.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+name: Execute tests on k8s - PowerPC
+on:
```
Comment (kishen-v) [inline .github/workflows/test-k8s-ppc64le.yml:2]: Awaiting inputs on the triggers for running tests.

**Comment by kishen-v** [comment]
Hi @krishvoor,
The latest run that had run successfully on my fork: https://github.com/kishen-v/kube-burner/actions/runs/10885156231.

Thank you!

File: .github/workflows/test-k8s-ppc64le.yml (hunk 1 above)
Comment (krishvoor) [reply .github/workflows/test-k8s-ppc64le.yml:2]: Can you add these steps to your steps into your workflow?
https://github.com/kube-burner/kube-burner/blob/main/.github/workflows/builders.yml#L20-L26

Comment (kishen-v) [reply .github/workflows/test-k8s-ppc64le.yml:2]: Addressed!
The latest run have the steps that are required to be added: https://github.com/kishen-v/kube-burner/actions/runs/10896532442/job/30236483958

**Review by vishnuchalla — APPROVED** [review]
lgtm

**Review by krishvoor — APPROVED** [review]
/lgtm

**Pull request merged**